packages/otelbin/src/components/validation-type/ValidationTypeContent.tsx

```diff
@@ -33,12 +30,9 @@ export default function ValidationTypeContent({
 				{currentDistro && (
 					<Button
 						onClick={() => {
-							router.push(
-								getUrl({
-									distro: "",
-									distroVersion: "",
-								})
-							);
+							if (typeof window !== "undefined") {
+								window.history.pushState(null, "", getLink({ distro: "", distroVersion: "" }));
```

Comment:
It is super weird that `router.push(…)` swallows these calls. We should look into the actual reason why Next.js is misbehaving in those cases.

Reply:
Actually, this is my question too.